manage.py

```diff
@@ -12,6 +22,18 @@ def cli():
 @cli.command()
 def init():
     Base.metadata.create_all(engine)
+    User._bootstrap()
+    sess = Session()
+    for user in sess.query(User).all():
+        Account._bootstrap(user)
+        Address._bootstrap(user)
+        Education._bootstrap(user)
+        Employment._bootstrap(user)
+        Language._bootstrap(user)
+        Profile._bootstrap(user)
+        Telephone._bootstrap(user)
+        Vitals._bootstrap(user)
```

Comment:
You should move each _bootstrap method into a function within a seeds.py file or file for each class following a './seeds/class.py' where class is the name of the class being seeded.